config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+  get 'users/show'
```

Comment:
remove line 2, 3. It's ready defined at line 10 `resources :users, only: %i(new create show)`